lib/mongodb.js

```diff
@@ -39,6 +40,7 @@ function ObjectID(id) {
   }
 }
 
+exports.generateMongoDBURL = generateMongoDBURL;
```

Comment:
In that case, can we replace https://github.com/elektron-technology/loopback-connector-mongodb/blob/master/lib/mongodb.js#L74 with `dataSource.connector = MongoDB(s, dataSource);`?